### ISSUE TITLE
[Feature Request]: Diagnostics without count and icon

### ISSUE THREAD

**Issue by vanarok** [akinsho/bufferline.nvim#452]
### What?

Minimal diagnostics only undercurl or underline and color, no indicators.

**with diagnostics and a gap:**
![2022-06-16T23:53:36,673431986+04:00](https://user-images.githubusercontent.com/6105977/174153244-9aedf8d5-bd76-4deb-b1cb-f851f6cc4460.png)

**with diagnostics and !**
![2022-06-16T23:52:36,325652314+04:00](https://user-images.githubusercontent.com/6105977/174153256-edc47cf3-33da-46e3-9c1a-67c5c5d285a7.png)

**without diagnostics:**
![2022-06-16T23:53:14,450769776+04:00](https://user-images.githubusercontent.com/6105977/174153253-f0cf1e9b-a34b-4ca4-8760-9afcd7b85b51.png)

### Why?

Minimalism. Don't bloat the tabs with useless junk, there are better places for diagnostic information. You can use " " instead of the icon, but it will take up extra space in the tab.

**Comment by akinsho** [comment]
@vanarok by default this plugin doesn't render any signs and just shows a count e.g. `(3)` if you feel this is *useless bloat*... you can just use
```lua
diagnostics_indicator = function(count, level, diagnostics_dict, context)
 return ""
end,
```
If that doesn't work for you, then I'd happily accept a PR

**Comment by vanarok** [comment]
> @vanarok by default this plugin doesn't render any signs and just shows a count e.g. `(3)` if you feel this is _useless bloat_... you can just use
> 
> ```lua
> diagnostics_indicator = function(count, level, diagnostics_dict, context)
>  return ""
> end,
> ```
> 
> If that doesn't work for you, then I'd happily accept a PR
 Your example disables diagnostics including underline and color.

**Comment by akinsho** [comment]
> I'd happily accept a PR

☝🏿 If you have a look at the `diagnostics.lua` module, you can probably change the behaviour of to allow highlighting but remove the indicator function. I'd recommend starting there

**Comment by akinsho** [comment]
Fixed by #460 which is in the `dev` branch but will get merged to main in the next release
